src/overlays/actors/ovl_En_Bx/z_en_bx.c

```diff
@@ -21,13 +28,216 @@ const ActorInit En_Bx_InitVars = {
     (ActorFunc)EnBx_Update,
     (ActorFunc)EnBx_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Bx/EnBx_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Bx/EnBx_Destroy.s")
+static ColliderCylinderInit sCylinderInit = {
+    { COLTYPE_UNK6, 0x11, 0x09, 0x00, 0x00, COLSHAPE_CYLINDER },
+    { 0x01, { 0xFFCFFFFF, 0x03, 0x04 }, { 0xFFCFFFFF, 0x01, 0x00 }, 0x01, 0x01, 0x00 },
+    { 60, 100, 100, { 0, 0, 0 } },
+};
+
+static ColliderQuadInit sQuadInit = {
+    { COLTYPE_UNK10, 0x11, 0x00, 0x00, 0x00, COLSHAPE_QUAD },
+    { 0x00, { 0xFFCFFFFF, 0x03, 0x04 }, { 0x00000000, 0x00, 0x00 }, 0x01, 0x00, 0x00 },
+    { { { 0.0f, 0.0f, 0.0f }, { 0.0f, 0.0f, 0.0f }, { 0.0f, 0.0f, 0.0f }, { 0.0f, 0.0f, 0.0f } } },
+};
+
+Vec3f D_809D250C = { 0.015f, 0.015f, 0.015f };
+
+Vec3f D_809D2518 = { 0.0f, 0.0f, 0.0f };
+
+static InitChainEntry sInitChain[] = {
+    ICHAIN_F32(unk_4C, 5300, ICHAIN_STOP),
+};
+
+Vec3f D_809D2528 = { 8000.0f, 15000.0f, 2500.0f };
+
+Vec3f D_809D2534 = { 8000.0f, 10000.0f, 2500.0f };
+
+Vec3f D_809D2540 = { -8000.0f, 15000.0f, 2500.0f };
+
+Vec3f D_809D254C = { -8000.0f, 10000.0f, 2500.0f };
+
+Color_RGBA8 D_809D2558 = { 255, 255, 255, 255 };
+Color_RGBA8 D_809D255C = { 200, 255, 255, 255 };
+
+UNK_PTR D_809D2560[] = { 0x060024F0, 0x060027F0, 0x060029F0, 0x00000000 };
```

Comment:
The last 0 word here is probably padding.
```suggestion
UNK_PTR D_809D2560[] = { 0x060024F0, 0x060027F0, 0x060029F0 };
```